hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueHandler.java

```diff
@@ -898,6 +900,7 @@ ContainerCommandResponseProto handleWriteChunk(
         blockDataProto = blockData.getProtoBufMessage();
         final long numBytes = blockDataProto.getSerializedSize();
         metrics.incContainerBytesStats(Type.PutBlock, numBytes);
+        metrics.incContainerOpsLatencies(Type.PutBlock, Time.monotonicNowNanos() - startTime);
```

Comment:
Is this already counted when cmdType is PutBlock here?

https://github.com/apache/ozone/blob/3096887c5c7f3c51d382b536e9fe0541ba0ab5fc/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java#L347

Reply:
No, it's counted under Type.WriteChunk because the RequestType is WriteChunk. We want to know how much is spent on PutBlock as well.